address/dynamodb.go

```diff
@@ -36,14 +37,14 @@ func NewDynamoDBResolver(client *dynamodb.DynamoDB, tableName string) Repository
 }
 
 func (r *dynamoDbResolver) Update(info *ResolveInfoType, routing, publicKey string) (bool, error) {
-	serial := strconv.Itoa(rand.Int())
+	serial := strconv.FormatUint(uint64(time.Now().Unix()), 10)
```

Comment:
This will generate the same serial for every client on each second.. we probably do not want this

Reply:
I guess using UnixNano should fix it?

Reply:
In theory (because it's lambda, and can run multiple at the same time), we can still get the same serials. Even though we can get that same issue with rand, I think it will be less likely. I think that using rand in this case mitigates any timing attacks that may happen (because we put the user basically in charge of generating the serial number)

Reply:
You’re right that it may be repeated but unlikely. However the reason why I want to use a timestamp is because in the future when we sync two key servers we may need a timestamp to resolve a conflict. 

We may use a random serial like before and add an internal timestamp to the record as well. Don’t you think?

Reply:
ok.. let's do UnixNano for  now I think